sockets/sockets_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
++// +build !windows,!appengine
```

Comment:
Looks like there's a stray `+` here
```suggestion
// +build !windows,!appengine
```